kyo-prelude/shared/src/main/scala/kyo/Parse.scala

```diff
@@ -499,41 +773,13 @@ object Parse:
     // ** Standard parsers **
     // **********************
 
-    /** Selects shortest matching parse from alternatives
-      *
-      * @param seq
-      *   Sequence of parsers to try
-      * @return
-      *   Result that consumed least input
-      */
-    def shortest[A: Flat, S](seq: (A < (Parse & S))*)(using Frame): A < (Parse & S) =
-        select(seq*) {
-            case (curr @ (state1, _), next @ (state2, _)) =>
-                if state2.position < state1.position then Maybe(next)
-                else Maybe(curr)
-        }
-
-    /** Selects longest matching parse from alternatives
-      *
-      * @param seq
-      *   Sequence of parsers to try
-      * @return
-      *   Result that consumed most input
-      */
-    def longest[A: Flat, S](seq: (A < (Parse & S))*)(using Frame): A < (Parse & S) =
-        select(seq*) {
-            case (curr @ (state1, _), next @ (state2, _)) =>
-                if state2.position > state1.position then Maybe(next)
-                else Maybe(curr)
-        }
-
     /** Consumes whitespace characters
       *
       * @return
       *   Unit after consuming whitespace
       */
-    def whitespaces(using Frame): Unit < Parse =
```

Comment:
I had made a few methods return `Unit` so users wouldn't need to discard values manually. We're now discarding values in the APIs themselves like in `Parse.between` and `andThen` works even for non-`Unit` computations so it's more ergonomic to return values.